### DISCUSSION TITLE
Windowing error on Sentinel-2

### DISCUSSION THREAD

**Discussion by bastiencerr** [sertit/eoreader#98] [Q&A] [answered]
Hello, 

I am using your library and i'm trying to crop my S2Product. I've followed your [tutorial](https://github.com/sertit/eoreader/blob/main/docs/notebooks/windowed_reading.ipynb), however it is based on only one band with a 10m resolution. I've noticed that if you try to crop on a index or a band with a lower resolution, such as SWIR_1 or NDBI, the `load` function returns an error : 

Code : 
``` 
def load(prod, band = [SWIR_1], window):
    ds = prod.load(band, window=window)
    prod.clean_tmp()
    return ds
```
Input : 
window = `POLYGON ((654378.8125000141 4820465.999998967, 666468.3125000218 4823420.499998713, 666701.562500022 4810358.999998731, 650958.0000000123 4810786.499999033, 651502.2500000127 4811136.499999024, 651502.2500000127 4811136.499999024, 654378.8125000141 4820465.999998967))` -> stored in a GeoDataFrame
source image =  `S2A_MSIL2A_20220716T103641_N0400_R008_T31TFJ_20220716T183414.SAFE`

Error : 
``` 
 File "<__array_function__ internals>", line 200, in where
ValueError: operands could not be broadcast together with shapes (1,655,789) (1,1310,1578) ()
``` 

If i launch this function with the RED band, for example, it works by the way

Thanks !

**Comment by remi-braun** [comment]
Hello,

This is weird, I cannot reproduce this issue.
Which version of eoreader are you using ?

**Reply by bastiencerr** [reply]
According to : `eoreader.__version__`
I have : 
**0.19.4**

**Reply by remi-braun** [reply, marked as answer]
Could you please test is again with the latest version (0.20.2) ? 😄

**Reply by bastiencerr** [reply]
Ok, it was the solution !! Thank you for the answer and sorry for the inconvenience !

**Reply by remi-braun** [reply]
No problem 🙏 Always glad to help !
